operations/configuration.md

```diff
@@ -115,6 +116,7 @@ Returns configuration of the enterprise and the client.
 | `Currencies` | array of [Accepted currency](#accepted-currency) | required | Currencies accepted by the enterprise. |
 | `Pricing` | string | required | [Pricing](#pricing) of the enterprise. |
 | `TaxPrecision` | string | optional | Tax precision used for financial calculations in the enterprise. If `null`, [Currency](currencies.md#currency) precision is used. |
+| `ExternalIdentifier` | string | optional, max 255 characters | Identifier of the enterprise from external system. |
```

Comment:
TBH this feels weird, external identifier of the enteprise. Any idea what is trully represent in enterprise?